lib/rubocop/cop/lint/constant_base.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Avoid meaningless `::` prefix on constant.
+      #
+      # @example
+      #   # bad
+      #   ::Const
+      #
+      #   # good
+      #   Const
+      #
+      #   # bad
+      #   class << self
+      #     ::Const
+      #   end
+      #
+      #   # good
+      #   class << self
+      #     Const
+      #   end
+      #
+      #   # good
+      #   class A
+      #     ::Const
+      #   end
+      #
+      #   # good
+      #   module A
+      #     ::Const
+      #   end
+      class ConstantBase < Base
+        extend AutoCorrector
+
+        MSG = 'Avoid meaningless `::` prefix on constant.'
+
+        # @param node [RuboCop::AST::CbaseNode]
```

Comment:
No param or return value need to be annotated like many other cops do. For example this `RuboCop::AST::CbaseNode` would not exist. Can you remove these annotations from this PR?